tchannel-core/src/main/java/com/uber/tchannel/handlers/RequestRouter.java

```diff
@@ -251,11 +251,14 @@ public void onFailure(Throwable e) {
             private void doRequestEndProcessing() {
                 if (span != null) {
                     span.finish();
+                    TracingContext tracingContext = topChannel.getTracingContext();
```

Comment:
Do you mean that `listeningExecutorService` is instrumented to propagate tracing context? Where is it initialized?